www/include/configuration/configServers/DB-Func.php

```diff
@@ -1075,3 +1075,25 @@ function checkChangeState(int $poller_id, int $last_restart): bool
 
     return false;
 }
+
+/**
+ * Define LocalPoller as Default Poller if there is no Default Poller
+ *
+ * @return void
+ */
+function defineLocalPollerToDefault() 
+{
+    global $pearDB;
+    $defaultPoller = [];
+    $query = "SELECT `is_default` FROM `nagios_server`";
```

Comment:
It would be better to count the number of servers like this:
`SELECT COUNT(*) FROM `nagios_server` where is_default = "1"`

Reply:
Resolve with commit 54adafd